validation/standard-config.toml

```diff
@@ -0,0 +1,9 @@
+# Standard Chain Config Parameters
+[resource_config]
+
```

Comment:
nit to remove the blank line
```suggestion
```

Reply:
[323a96b](https://github.com/ethereum-optimism/superchain-registry/pull/185/commits/323a96b095dc46328cdafdecdbf64091cf8d970b)